### ISSUE TITLE
Network aliases are not respected (Docker for Mac M1)

### ISSUE THREAD

**Issue by kblcuk** [docker/compose-cli#1713]
[`aliases`](https://docs.docker.com/compose/compose-file/compose-file-v3/#aliases) specified for network are not being respected at least when running Compose CLI on Mac.

You can use the following minimal example to reproduce it:

`docker-compose.yml`
```
version: '3'

  services:
    box:
      image: alpine:latest
      container_name: box
      tty: true
      networks:
        test:
          aliases:
            - whoami.dev
            - another-alias.local

  networks:
    test:
      name: test-net
```

Then, running it and inspecting assigned aliases gives:
```
❯ docker compose up
[+] Running 3/2
 ⠿ Network test-net                  Created                                                                                       3.8s
 ⠿ Network proxy-playground_default  Created                                                                                       3.9s
 ⠿ Container box                     Created                                                                                       0.0s
Attaching to box
```
```
(in another shell)
❯ docker inspect box --format='{{range .NetworkSettings.Networks}}{{.Aliases}}{{end}}'
[box box 70e52834a01e]
```

Expected output can be achieved by using stand-alone `docker-compose`:
```
❯ docker-compose up
Docker Compose is now in the Docker CLI, try `docker compose up`

Recreating box ... done
Attaching to box
```
```
(in another shell)
❯ docker inspect box --format='{{range .NetworkSettings.Networks}}{{.Aliases}}{{end}}'
[b1545c857922 another-alias.local whoami.dev box]
```

I couldn't reproduce this on my Manjaro box, so it might be that this is Mac (M1?) specific.
Docker version gives me the following:
```
❯ docker version
Client:
 Cloud integration: 1.0.14
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.16.3
 Git commit:        370c289
 Built:             Fri Apr  9 22:46:57 2021
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.6
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8728dd2
  Built:            Fri Apr  9 22:44:13 2021
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.4.4
  GitCommit:        05f951a3781f4f2c1911b05e61c160e9c30eaa8e
 runc:
  Version:          1.0.0-rc93
  GitCommit:        12644e614e25b05da6fd08a38ffa0cfe1903fdec
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
And I'm running Docker for Mac `3.3.3 (64133)`.

Let me know if you need more info! 😊

**Comment by ndeloof** [comment]
This issue has been fixed already in 1.0.15

**Comment by kblcuk** [comment]
Right, I somehow missed https://github.com/docker/compose-cli/issues/1620 during my search for similar issues.
Sorry for the dupe report.
